rasa/core/policies/policy.py

```diff
@@ -435,6 +442,7 @@ def __eq__(self, other: Any) -> bool:
             and self.events == other.events
             and self.optional_events == other.events
             and self.is_end_to_end_prediction == other.is_end_to_end_prediction
+            # We do not compare `diagnostic_data`
```

Comment:
I think it would be better to describe the `why` as the `what` is quite clear from the code